content/index.ko.md

```diff
@@ -58,29 +59,31 @@ footer: >
 {% include_cached toc.html type="end" %}
 {:/}
 
-## 상황속 접근성 {#context}
+## 상황에 따른 접근성 {#context}
```

Comment:
@remibetin 
Based on your explanation, I think “Accessibility in Context” is an idiom in English(putting x in context) which is like an introduction to the concept of accessibility. So, “Accessibility in Context” can’t be translated directly but the closest meaning would be “웹 접근성의 이해 (Understanding web accessibility)” or “웹 접근성 소개 (Introduction to web accessibility),” but personally, I think “웹 접근성의 이해 (Understanding web accessibility)” is better, because people usually use the word “이해 understanding” in a situation like this in Korea.